### ISSUE TITLE
Closed products bullet in Scope section needs to be clarified

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#8]
In the WCAG2ICT Note from 2013, the [Excluded from Scope section](https://www.w3.org/TR/wcag2ict/#intro_excluded) has a bullet that is complex to read and could potentially be misunderstood. This should be clarified in the the update to the WCAG2ICT Note.

The current bullet reads:

> Because this document deals with applying WCAG, which is a standard for web content accessibility, to ICT it does not deal with such things as closed products and requirements for non-user interface aspects of platforms, nor individual components. As such, this document is not sufficient by itself to ensure accessibility in non-web documents and software.

**Comment by maryjom** [comment]
Suggest replacing two bullets that are very similar (quoted from the 2013 WCAG2ICT):

>- Because this document deals with applying WCAG, which is a standard for web content accessibility, to ICT it does not deal with such things as closed products and requirements for non-user interface aspects of platforms, nor individual components. As such, this document is not sufficient by itself to ensure accessibility in non-web documents and software.
>- This document does not comment on hardware aspects of products, non-user interface aspects of platforms, or user-interface components as individual items, because the basic constructs on which WCAG 2.0 is built do not apply to these.

with a single bullet that reads: 
-   This document is not sufficient by itself to ensure accessibility in non-web documents and software. As a web standard, WCAG does not fully cover all accessibility requirements for non-user interface aspects of platforms, user-interface components as individual items, nor closed product software (where there is no Assistive Technology to communicate programmatic information).

**Comment by maryjom** [comment]
I handled a little differently in the pull request to update the introductory sections. See PR #88 .
